cadc-rest/build.gradle

```diff
@@ -14,7 +14,7 @@ sourceCompatibility = 1.7
 
 group = 'org.opencadc'
 
-version = '1.1.1'
+version = '1.1.4'
```

Comment:
why the jump from 1.1.1 to 1.1.4?

Reply:
good question, I probably restarted work 3 times :-) Do you think 1.1.2 or 1.2 is more approp?

Reply:
Probably 1.2